doozerlib/assembly_inspector.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+
+from typing import List, Dict, Optional
```

Comment:
Would like to remove the shebang if this file is not intended to run directly.

```suggestion
from typing import List, Dict, Optional
```

Reply:
Done!